db/migrate/20230821123151_create_users.rb

```diff
@@ -1,19 +1,18 @@
 class CreateUsers < ActiveRecord::Migration[7.0]
-  def change
+  def up
     create_table :users do |t|
-      #primary key gets created automaticly
-      t.string "fist_name", :limit => 25
-      t.string "last_name", :limit => 25
+      t.string "fist_name"
+      t.string "last_name"
       t.string "role", :default => 'user'
```

Comment:
lets introduce enums in a different PR and not here therefore lets remove the default and keep it a string only
```suggestion
      t.string "role"
```